root/etc/profile.d/discovery.sh

```diff
@@ -0,0 +1,8 @@
+# /etc/profile.d/discovery.sh - shell extensions for discovery
+
+# Changes to this file should also be reflected in the setup-extension systemd
+# service file, as systemd doesn't source this file
+
+export LD_LIBRARY_PATH=/opt/extension/lib
+export RUBYLIB=/opt/extension/lib/ruby
+export PATH=$PATH:/opt/extension/bin
```

Comment:
Maybe you want to prepend instead of append so users can overwrite system commands.